controllers/keyword.go

```diff
@@ -136,10 +146,27 @@ func getKeywordsData(keywordService keyword_service.KeywordService) map[string]i
 
 	data := getCurrentUser(keywordService)
 	data["keywordPresenters"] = keywordPresenters
+	data["filter"] = queryString
 
 	return data
 }
 
+func validateValidConditions(queryString map[string][]string) []models.Condition {
```

Comment:
@gutakk How about changing the function name to `filterValidConditions` or `validConditions` so it's more aligned with the function behavior, IMO naming it `validate` sound like it would return boolean or add some error if the input does not pass the validation 🤔

Reply:
Indeed it could be a good naming improvement. Just like in Rails there are strong params to permit only certain params.

Reply:
Good point! Fixed in 2060a940e9d85e9b31d4d90a3e92a4539b2b4268